packages/taco/src/conditions/schemas/jwt.ts

```diff
@@ -0,0 +1,21 @@
+import { z } from 'zod';
+
+import { baseConditionSchema } from './common';
+import { contextParamSchema } from './context';
+
+export const JWT_PARAM_DEFAULT = ':jwtToken';
+
+export const JWTConditionType = 'jwt';
+
+export const jwtConditionSchema = baseConditionSchema.extend({
+  conditionType: z.literal(JWTConditionType).default(JWTConditionType),
+  publicKey: z.string().optional(),
```

Comment:
Based on https://github.com/nucypher/nucypher/pull/3570, `publicKey` is a required property.

Reply:
✔️